### PR TITLE
Class basic syntax

### PR DESCRIPTION
**Pull request by Amirhossein-Veysi**

### CODE REVIEW

File: 1-js/09-classes/01-class/article.md [hunk 1]
```diff
@@ -50,87 +50,89 @@ user.sayHi();
...
+alert(User.prototype.sayHi); // sayHi کد متد
 
-// there are exactly two methods in the prototype
-alert(Object.getOwnPropertyNames(User.prototype)); // constructor, sayHi
+// دقیقا دو متد در پروتوتایپ وجود دارد
+alert(Object.getOwnPropertyNames(User.prototype)); // sayHi ،سازنده
 ```
 
-## Not just a syntactic sugar
+## فقط یک سینتکس شکری نیست
```
Comment (Amirhossein-Veysi) [inline 1-js/09-classes/01-class/article.md:121]: I'm not sure about this translation, Let me know if it's wrong.

Comment (mahdyar) [reply 1-js/09-classes/01-class/article.md:121]:
```suggestion
## فقط یک سینتکس برای زیباکردن(syntactic sugar) نیست
```

**Review by mahdyar — CHANGES_REQUESTED** [review]
Please review the suggestion(s).

**Comment by Amirhossein-Veysi** [comment]
Thanks @mahdyar. /done

File: 1-js/09-classes/01-class/1-rewrite-to-class/task.md [hunk 2]
```diff
@@ -2,8 +2,8 @@ importance: 5
...
 ---
 
-# Rewrite to class
+# با کلاس بازنویسی کنید
 
-The `Clock` class (see the sandbox) is written in functional style. Rewrite it in the "class" syntax.
+کلاس `Clock` (به جعبه شنی مراجعه کنید) به سبک تابع نوشته شده است. آن را در سینتکس "کلاس" بازنویسی کنید.
 
-P.S. The clock ticks in the console, open it to see.
+ضمیمه: تیک تاک ساعت در کنسول، آن را باز کنید تا ببینید.
```
Comment (mahdyar) [inline 1-js/09-classes/01-class/1-rewrite-to-class/task.md:9]:
```suggestion
ضمیمه: ساعت تیک‌تاک می‌کند در کنسول، آن را باز کنید تا ببینید.
```

File: 1-js/09-classes/01-class/article.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
 
-# Class basic syntax
+# سینتکس پایه کلاس
 
-```quote author="Wikipedia"
-In object-oriented programming, a *class* is an extensible program-code-template for creating objects, providing initial values for state (member variables) and implementations of behavior (member functions or methods).
+```quote author="ویکی پدیا"
```
Comment (mahdyar) [inline 1-js/09-classes/01-class/article.md:4]:
```suggestion
```quote author="ویکی‌پدیا"
```

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Review by mahdyar — APPROVED** [review]
Thank you, @Amirhossein-Veysi!

**Pull request merged**